pyproject.toml

```diff
@@ -74,8 +74,8 @@ dependencies = [
     "timm>=0.4.12",
     # torch 1.13+ required by torchvision
     "torch>=1.13",
-    # torchmetrics 0.10+ required for binary/multiclass/multilabel classification metrics
-    "torchmetrics>=0.10",
+    # torchmetrics 1.1.1+ required for average argument to MeanAveragePrecision
```

Comment:
According to https://github.com/Lightning-AI/torchmetrics/commit/63c7bbe6ac163659e9631badce296ff1c658d45e the argument didn't exist until 1.2